Extractor/UAExtractor.cs

```diff
@@ -95,6 +95,10 @@ public class UAExtractor : BaseExtractor<FullConfig>, IUAClientAccess
 
         private readonly ILogger<UAExtractor> log;
 
+        private readonly RebrowseTriggerManager? serverSubscription;
```

Comment:
Rename the manager here as well, just `rebrowseTriggerManager` or something is fine.